src/redux/Books/BooksSclice.js

```diff
@@ -1,10 +1,28 @@
 import { createSlice } from '@reduxjs/toolkit';
 
+const initialState = [
+  {
+    item_id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item3',
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  },
+];
 const booksSlice = createSlice({
   name: 'books',
-  initialState: {
-    list: [],
-  },
+  initialState,
   reducers: {
     addBook: (state, action) => {
       state.list.push(action.payload);
```

Comment:
- [ ] you're not exporting the  `bookSclice` as default so it returns an error when you run the app please make sure to fix that in order to review your app
<img width="908" alt="Screenshot 2023-03-07 at 14 39 37" src="https://user-images.githubusercontent.com/61790587/223439794-6a74d58b-d6e5-493a-bd95-648e3ddc0e1b.png">